tetrateci/test_1.7.sh

```diff
@@ -26,7 +26,7 @@ if [[ ${CLUSTER} == "eks" ]]; then
   git apply tetrateci/patches/eks/eks-ingress.1.7.patch
 fi
 
-PACKAGES=$(go list ./tests/integration/... | grep -v /qualification | grep -v /examples | grep -v /multicluster)
+PACKAGES=$(go list ./tests/integration/... | grep -v /qualification | grep -v /examples | grep -v /multicluster | grep -v /stackdriver)
```

Comment:
-tags=integ -- is it not needed here?

Here and elsewhere can we use egrep instead? Something like
egrep -v '/qualification|/examples|/multicluster|/stackdriver'

Reply:
Apparently the tag should be there but it was not added, I have proposed a PR to upstream istio, until then the tag would include a broken test (which I too fixed in that PR).